packages/kbn-check-mappings-update-cli/current_mappings.json

```diff
@@ -1783,6 +1783,9 @@
           }
         }
       },
+      "supports_agentless": {
+        "type": "boolean"
+      },
```

Comment:
Q: do we need the field in the mappings? In other words: are we planning to search/aggregate by this field?

Reply:
we do indeed. this PR is already using it to filter agentless vs non-agentless policies:
https://github.com/elastic/kibana/blob/605fadc54c6bb19a3a392524cde41617dadbbe9d/x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/detail/policies/package_policies.tsx#L146